fixtures/Entity/DummyWithPrivateProperty.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of the Alice package.
+ *
+ * (c) Nelmio <hello@nelm.io>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Nelmio\Alice\Entity;
+
+class DummyWithPrivateProperty
+{
+    private $val;
```

Comment:
You mean to ensure the accessor never consider static properties? Done (and accessor updated).